tests/functions/test_map_to_s3.py

```diff
@@ -0,0 +1,134 @@
+from cdn_lambda.functions.map_to_s3 import lambda_handler
+import os
+import mock
+import json
+
+
+@mock.patch("boto3.client")
+@mock.patch("cdn_lambda.functions.map_to_s3.datetime")
+def test_map_to_s3(mocked_datetime, mocked_boto3_client):
+    mocked_datetime.utcnow.return_value = "2020-02-12 20:55:06.372878"
+    mocked_boto3_client().query.return_value = {
+        "Items": [
+            {
+                "web_uri": {"S": "www.example.com/content/file.ext"},
+                "from_date": {"S": "2020-02-12 21:02:59.097169"},
+                "object_key": {"S": "e4a3f2sum"},
+            }
+        ]
+    }
+
+    env_vars = {"DB_TABLE_NAME": "test_table", "DB_TABLE_REGION": "us-east-1"}
+    event = {
+        "Records": [
+            {"cf": {"request": {"uri": "www.example.com/content/file.ext"}}}
+        ]
+    }
+
+    with mock.patch.dict(os.environ, env_vars):
```

Comment:
There's actually a built-in fixture to help with this - https://docs.pytest.org/en/latest/monkeypatch.html

```
monkeypatch.setenv("DB_TABLE_NAME", "test_table")
monkeypatch.setenv("DB_TABLE_REGION", "us-east-1")
```

They'll be unwound automatically at the end of the test.  I leave you to decide if this is better or worse than the current approach.

Reply:
I think I prefer the current approach as it does the same thing fewer imports and less code.